src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -104,7 +106,9 @@ const ResourceList = () => {
     const isSuperAdmin = !!userRole?.result.superAdmin
 
     const isOverviewNodeType = nodeType === SIDEBAR_KEYS.overviewGVK.Kind.toLowerCase()
+    const isMonitoringNodeType = nodeType === SIDEBAR_KEYS.monitoringGVK.Kind.toLowerCase()
     const isTerminalNodeType = nodeType === AppDetailsTabs.terminal
+    const fixedTabIndices = getFixedTabIndices()
```

Comment:
Can't directly move to constants since was getting error that importComponentFromFeLib was not present at that moment